### DISCUSSION TITLE
Has anyone here ever used ssh tunneling for whatsmeow?

### DISCUSSION THREAD

**Discussion by rvp98** [tulir/whatsmeow#241] [General]
I need to use SSH Tunelling because my Postgresql hosting need to use that for remote the database

**Comment by Manjit2003** [comment]
You don't need ssh tunneling for this. Just host your db on another server and open the db port. You can easily connect to the db using the connection string. Also make sure to have a strong password.

**Reply by rvp98** [reply]
Im sorry, i need the ssh tunelling because of my hosting server is only allow connection with ssh tunelling 

I already try without ssh tunelling but, i get wrong password error

**Comment by mamur-rezeki** [comment]
look at this https://github.com/jackc/pgx/issues/382

**Reply by rvp98** [reply]
Is whatsmeow support with pgx sir?

**Comment by mamur-rezeki** [comment]
Yes, according to this merge
https://github.com/tulir/whatsmeow/pull/183/commits/efd0d7795c7953f64355d08b8241dc6753ac08a2
